woocommerce/pinterest-for-woocommerce
# Remove unnecessary padding in `Catalog > Overview` table

### Changes proposed in this pull request

Version tested: c7d3c3f70fcfba6fa220da2e1a17a5e47b076df5

This PR removes the padding in the Overview section.

Previously we have `CardBody` with `no-padding` CSS class, however this seems to be overwritten by `@wordpress/components` generated style: 

![image](https://user-images.githubusercontent.com/417342/128402843-07fb8aaf-2cfa-4c9d-be39-4cddd273533a.png)

We can remove the padding by not using `CardBody`, which is implemented in this PR.

#### Screenshots

Prior to this PR, we have padding inside Overview: 

![image](https://user-images.githubusercontent.com/417342/128402227-f965dbdd-eddf-4371-ad0c-c74ba9477853.png)

With this PR, the padding is removed:

![image](https://user-images.githubusercontent.com/417342/128402160-d34136f2-50e8-49a7-90df-2064fad47dc7.png)

### Detailed test instructions

1. Go to Catalog page: /wp-admin/admin.php?page=wc-admin&path=%2Fpinterest%2Fcatalog
2. Make sure there is no padding in Overview section.

### Changelog note

Fix: Remove padding in Overview section.
